mempool/src/feerate.rs

```diff
@@ -0,0 +1,72 @@
+use std::num::NonZeroU128;
+
+use common::primitives::amount::Amount;
+
+lazy_static::lazy_static! {
+    pub(crate) static ref INCREMENTAL_RELAY_FEE_RATE: FeeRate = FeeRate::new(Amount::from_atoms(1000));
+    pub(crate) static ref INCREMENTAL_RELAY_THRESHOLD: FeeRate = FeeRate::new(Amount::from_atoms(500));
+}
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+pub(crate) struct FeeRate {
+    atoms_per_kb: u128,
+}
+
+impl FeeRate {
+    pub(crate) fn new(tokens_per_kb: Amount) -> Self {
+        Self {
+            atoms_per_kb: tokens_per_kb.into_atoms(),
+        }
+    }
+
+    pub(crate) fn of_tx(fee: Amount, tx_size: usize) -> Self {
```

Comment:
- I would call this `from_total_tx_fee()`
- I would call the first argument `total_tx_fee`

Reply:
Cool. Done.